index.bs

```diff
@@ -4036,7 +4322,10 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         "hybridTransport",
         "passkeyPlatformAuthenticator",
         "userVerifyingPlatformAuthenticator",
-        "relatedOrigins"
+        "relatedOrigins",
+        "signalAllAcceptedCredentials",
+        "signalCurrentUserDetails",
+        "signalUnknownCredential"
```

Comment:
(Continuing the discussion from https://github.com/w3c/webauthn/pull/2093#issuecomment-2253419585 and https://github.com/w3c/webauthn/pull/2093#pullrequestreview-2197576198)

I can see one argument against these: it adds an opportunity for implementation bugs. A client _could_ implement the functions but forget to report (some of) these capabilities, or vice versa. It's probably not very _likely_ that that would happen, but I think RPs can feature-detect the functions perfectly well without these capabilities - checking for the existence of types and functions is fairly normalized among web APIs. So why not eliminate the risk entirely when we can?

Reply:
I'm good either way. We already have `conditionalGet` and `userVerifyingPlatformAuthenticator` which have their own way to be feature detected as well. I don't think it's terribly important whether we add these or not.

Reply:
Let's try and keep `getClientCapabilities()` up-to-date as we continue adding features, it'd be a shame if it falls out of sync so early after getting into L3.

In this PR I'd recommend adding three new capabilities, one for each of the new `signal*` methods.

Reply:
Oh, you've already implemented these three new capabilities. Thank you! 🎉